### ISSUE TITLE
[Classic] Quest: Burning Blade Medallion

### ISSUE THREAD

**Issue by GnaXi** [cmangos/issues#1746]
**Current behavior**: 
Cannot pick up quest item: Burning Blade Medallion.

**Expected behavior**: 
To be able to pick it up.

**Steps to reproduce**:
1. Kill Yarrog Baneshadow
2. Try to pick up Burning Blade Medallion
3. Fail to pick it up

**Client version**:
1.12.1

**Commit hash**: 
8c5be3b665b6a3f133261d2e675f19a47330c5dc

**Database version**: 
Classic DB version 1.9 "The Black Dragonflight". For core Classic z2717.

**Operating system**: 
Debian 9

**Comment by AnonXS** [comment]
cant confirm, works on tbc-db

**Comment by Grz3s** [comment]
wotlkDB also works perfect

**Comment by cyberium** [comment]
Ok its replicable but you have to be in group.

```diff
diff --git a/src/game/Loot/LootMgr.cpp b/src/game/Loot/LootMgr.cpp
index fdbce2cfc..7312c4c5f 100644
--- a/src/game/Loot/LootMgr.cpp
+++ b/src/game/Loot/LootMgr.cpp
@@ -945,7 +945,7 @@ bool Loot::FillLoot(uint32 loot_id, LootStore const& store, Player* lootOwner, b
         for (auto lootItem : m_lootItems)
         {
             // roll for over-threshold item if it's one-player loot
-            if (!lootItem->freeForAll && lootItem->itemProto->Quality < uint32(m_threshold))
+            if (lootItem->freeForAll || lootItem->itemProto->Quality < uint32(m_threshold))
                 lootItem->isUnderThreshold = true;
             else
             {
```
That patch should fix the issue. I'll test it more tomorrow and push it if its ok

**Comment by GnaXi** [comment]
Yeah. I was using playerbots. So I was in a group.
Should probably have mentioned that. Sorry! :)

**Comment by jimmybrancaccio** [comment]
This should now be resolved in cmangos/mangos-classic@c9d36e2. Please open a new issue if you're still experiencing any issues. Thanks!
